Packages/org.plunderludics.UnityHawk/Runtime/Emulator/Emulator.Api.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace UnityHawk {
 
-public partial class Emulator : MonoBehaviour
+public partial class Emulator
```

Comment:
partial class only needs to inherit in one place (so you could have like 3 files, each implements an interface, without having the clutter up here), i might be too partial class pilled from discone, and think this might as well be a Api.cs at some point.

Reply:
Ok makes sense, i like the partial class thing is great here fwiw, seems nicer than having a separate api object